duckduckgo/duckduckgo-privacy-extension
# Fingerprint protection

**Reviewer:** Konrad

## Description:
Adding basic fingerprinting protection to the extension. This will prevent websites from gleaning screen, battery, and some system information provided by the window, navigator, battery, and screen API's.


## Steps to test this PR:
1. Run automated tests with npm run test-int
2. Run manual tests by installing the plugin and going to any page. Open dev tools, and check values of window.availTop or availLeft (should be 0).

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
